out/contracts.json

```diff
@@ -806,10 +806,283 @@
     "osmo-test-5": [
       [
         {
-          "protocol": "croncat",
-          "contract": "factory"
```

Comment:
Hm, it overrides old values, not sure how we want to deal with it, if the list of "don't override" is low we can probably hardcode it

Reply:
In the network itself you can add manual contract entries, similar to the asset entries added in `networks/osmosis1.ts`

Reply:
It's good that it overrides them

Reply:
> It's good that it overrides them

It is, but some of them we would like to keep, I assume